mobile/lib/modules/backup/services/backup.service.dart

```diff
@@ -79,7 +117,13 @@ class BackupService {
         backupAlbums.lastExcludedBackupTime,
         now,
       );
-      return toAdd.toSet().difference(toRemove.toSet()).toList();
+      final Set<String> duplicatedAssetIds = getDuplicatedAssetIds();
```

Comment:
@zoodyy I added this in to get the upload candidate for the background upload. I like your naming and the cleanliness of this function. I haven't given it much thoughts but can we use this method for the foreground upload as well?

Reply:
yes, we can also use those methods for the foreground backup.
However, it won't help with showing the assets stats (total, remaining etc.)